docs/demos/scripts/bootstrap.script

```diff
@@ -58,7 +58,7 @@ CLEAR
 ### STEP 1: download EasyBuild bootstrap script
 ###
 
-curl -O COPYPASTEhttps://raw.githubusercontent.com/hpcugent/easybuild-framework/develop/easybuild/scripts/bootstrap_eb.py
+curl -O COPYPASTEhttps://raw.githubusercontent.com/easybuilders/easybuild-framework/develop/easybuild/scripts/bootstrap_eb.py
```

Comment:
No, this is correct, the `COPYPASTE` is a keyword that tells my patched version of `asciicinema` to copy-paste what follows. Adding a space after would also add a space in what is being copy-pasted.